### DISCUSSION TITLE
Mimic a system Programmers visual trouble shooting actions.

### DISCUSSION THREAD

**Discussion by sunil390** [IBM/tnz#148] [Ideas]
I have an idea... let's assume we want to mimic how a system programmer will login to mainframe open an stc/job log and then do visual search to identify problem patterns to resolve an issue. How we can combine tnz to combine with available models in huggingface/transformers to exactly mimic this

1. Give an input as an stc name or job name that need to be analyzed.
2. Through python ati scripting logon to mainframe and capture screen outputs.
3. Pass on these text outputs to LLM models for continuous inference to get problem patterns and insights.
4. Pass on insights to a system programmer through a post on a teams chat.

Any suggestions tweaks to this idea is welcome 😊

**Comment by v1gnesh** [comment]
This depends on each site's logon process (ex: session manager).
So it's not possible to write a generic screen automation script to do so.
However, a more direct script with REXX/python should be possible.
